src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -382,8 +385,25 @@ def showHelp(self, url):
         else:
             url_abs = Path(url)
         try:
-            # Help window shows itself
-            self.helpWindow = DocViewWindow(parent=self, source=url_abs)
+            # In order to have multiple help windows open simultaneously, we need to create a new class variable
+            # If we just reassign the old one, the old window will be destroyed
+            name_found = False # Have we found an available class variable to assign the help window to?
+            while name_found is False:
+                potential_help_window = getattr(cls, window_name, None) # A potential variable
+                if potential_help_window == None:
+                    # Create a new help window
+                    name_found = True
+                elif potential_help_window.isVisible():
+                    window_name = f"help_window_{counter}"
+                    counter += 1
+                    continue
+                else:
+                    name_found = True
```

Comment:
I think this simplifies to:

```suggestion
            # Have we found a name not assigned to a window?
            potential_help_window = getattr(cls, window_name, None) 
            while potential_help_window:
                window_name = f"help_window_{counter}"
                potential_help_window = getattr(cls, window_name, None)
                counter += 1
```